x-pack/plugins/security_solution/server/plugin.ts

```diff
@@ -223,19 +223,59 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
         insightsAndAlerting: ['triggersActions'],
       },
       alerting: [SIGNALS_ID, NOTIFICATIONS_ID],
+      cases: [APP_ID],
+      subFeatures: [
+        {
+          name: 'Cases',
+          privilegeGroups: [
+            {
+              groupType: 'mutually_exclusive',
+              privileges: [
+                {
+                  id: 'cases_all',
+                  includeIn: 'all',
+                  name: 'All',
+                  savedObject: {
+                    all: [...caseSavedObjects],
```

Comment:
👍

Reply:
Oh I see. I thought that first the security plugin authorized the savedObject and after that we do our RBAC. Based on your previous comments it seems that our RBAC is enough as we use the security plugin to check for user privileges, right?

Reply:
Yeah since you're doing your own RBAC, there's no need for the security plugin to do anything here. This would actually weaken your security, as it could allow someone to get access to cases that are owned by a feature they aren't authorized to use.